packages/gatsby/gatsby-browser.js

```diff
@@ -1,6 +1,7 @@
 exports.onClientEntry = function(_, pluginParams) {
   require.ensure(['@sentry/react'], function(require) {
     const Sentry = require('@sentry/react');
+    const hasTracingEnabled = require('@sentry/utils').hasTracingEnabled;
```

Comment:
Can we make sure `@sentry/utils` is in `require.ensure`, and explicitly put it in the deps if we are going to use it?

Reply:
Sure. TIL about `require.ensure()`. LMK if there's more I need to do here, besides adding it to the array and to `package.json`.